README.md

```diff
@@ -18,35 +18,47 @@ The Ballerina Discord connector offers APIs to connect and interact with the [Di
 Follow these steps to create a Discord developer account.
 
 ### Step 1: Login to Discord developer page
-
+<br/>
```

Comment:
No need to add line break elements and having an indentation (similar to https://github.com/ballerina-platform/module-ballerinax-zendesk/blob/main/ballerina/Module.md?plain=1) should be enough.

Reply:
All changes made

Reply:
@KamalikaSene it seems this is still not addressed

Reply:
Changes have now been made

Reply:
@NipunaRanasinghe Changes have now been made.